spec/auth/controllers/auth/create_spec.rb

```diff
@@ -60,6 +60,10 @@
 
   after { user_repo.clear }
 
+  before do
+    expect(Workers::Auth::ImportRepos).to receive(:perform_async)
```

Comment:
just my 5 cents, but I prefer to use this way for testing workers in sidekiq
https://github.com/mperham/sidekiq/wiki/Testing#testing-worker-queueing-fake